src/functions.ts

```diff
@@ -62,7 +75,7 @@ export interface CloudFunctionsContext {
   /**
    * The resource that emitted the event.
    */
-  resource?: string;
+  resource?: string | object;
```

Comment:
👍 I noticed this was an issue in my last PR but was reluctant to touch it...

Reply:
👌🏻